doc/sphinx/source/notes_7_2.rst

```diff
@@ -0,0 +1,16 @@
+**************************
+Release Notes for PoCL 7.2
+**************************
+
+===========================
+Driver-specific features
+===========================
+
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+CUDA driver
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+* Implement sub_group_{reduce,scan_exclusive,scan_inclusive}_* and
+  sub_group_{all,any,broadcast}. (reduce is only available for PTX6.0+)
+* Note that CUDA driver does not support LLVM 21, due to a bug
+  in upstream Clang code. Users must use LLVM 17 to 20 with CUDA.
```

Comment:
Would be nice if there were a configure-time check for the unsupported combinations. 

Could you mention the bug in the `cuda.rst`'s known issues section with a link to the upstream issue?